CMakeLists.txt

```diff
@@ -544,6 +576,16 @@ psibase_package(
     DEPENDS wasm
 )
 
+psibase_package(
+    OUTPUT ${SERVICE_DIR}/SupervisorSys.psi
+    NAME SupervisorSys
+    VERSION ${PSIBASE_VERSION}
+    PACKAGE_DEPENDS NftSys ProxySys AccountSys
+    SERVICE supervisor-sys
+        DATA ${CMAKE_CURRENT_SOURCE_DIR}/services/user/SupervisorSys/ui/dist /
+    DEPENDS SupervisorSys_js ${CMAKE_CURRENT_SOURCE_DIR}/services/user/SupervisorSys/ui/dist/index.html
```

Comment:
`${SupervisorSys_js_dep}` would be better

Reply:
fixed